packages/components/src/Combobox/Combobox.test.tsx

```diff
@@ -329,3 +329,36 @@ describe("ComboboxContent Search", () => {
     expect(getByText('No results for "Bilbo"')).toBeInTheDocument();
   });
 });
+
+describe("Combobox Zero Index State", () => {
+  describe("when no options exist and contentNoun is provided", () => {
+    it("should render the correct zero index state text", () => {
+      const contentNoun = "teammates";
+      const { getByText } = render(
+        <Combobox>
+          <Combobox.TriggerButton label="Select a teammate" />
+          <Combobox.Content
+            options={[]}
+            onSelection={jest.fn()}
+            contentNoun={contentNoun}
+          />
+        </Combobox>,
+      );
+
+      expect(getByText("You don't have any teammates yet")).toBeInTheDocument();
+    });
+  });
+
+  describe("when no options exist and contentNoun is not provided", () => {
+    it("should render the default zero index state text", () => {
+      const { getByText } = render(
+        <Combobox>
+          <Combobox.TriggerButton label="Select a tax rate" />
+          <Combobox.Content options={[]} onSelection={jest.fn()} />
+        </Combobox>,
+      );
+
+      expect(getByText("No options yet")).toBeInTheDocument();
+    });
+  });
+});
```

Comment:
what do you think about one more test case, where we do have a > 0 length of options, and we assert that the `No options yet` should **not** be found?

Reply:
Yes - great idea!

Reply:
Done here - 5b612edce4bbc27891df8ecb55c74dfb70917cff